staff/daniel-toledo/gamma-template/style.sass

```diff
@@ -0,0 +1,419 @@
+    color: rgb(28,170,152)
+
+.header
+    background-color: rgb(27,34,42)
+    display: flex
+    flex-direction: column
+    justify-content: center
+
+    &__menu
+        background-color: rgb(248, 248, 248)
+        display: flex
+        flex-direction: row
+        justify-content: space-between
+        align-items: center
+
+    &__logo
+        padding: 10px
+        @extend %green
+
+    &__nav
+        display: none
+    
+    &__burguer
+        border: lightgrey 1px solid
+        border-radius: 0.25em
+        padding: 10px
+        margin-right: 30px
+    
+    &__page-title
+        background-color: rgb(28,170,152)
+
+    &__page-title h1
+        padding: 10px
+        margin-left: 20px
+        color: white
+        font-size: 40px
+        font-family: Arial, Helvetica, sans-serif
+    
+    &__title
+        margin: 1em
+        text-align: center
+        color: white
+    
+    &__title--green
+        @extend %green
+    
+    &__text
+        margin: 1em
+        text-align: center
+        color: white
+        font-family: Arial, Helvetica, sans-serif
+
+    &__images
+        display: flex
+        flex-direction: column
+        justify-content: center
+
+    &__img
+        margin: 1em auto
+        width: 80%
+        max-width: 300px
+    
+    &__picture
+        display: none
+
+.content
+    margin: 4em auto
+    padding: 0px 10px
+
+    &__img
+        width: 100%
+        border: solid lightgrey 1px
+
+    &__title
+        margin: 1em auto
+    
+    &__text
+        margin: 1em auto
+        font-family: Arial, Helvetica, sans-serif
+
+    &__btn
+        cursor: pointer
+        outline: 0
+        padding: 5px 10px
+        background-color: white
+        border: solid lightgrey 1px
+        border-radius: 0.3em
+     
+    &__btn:hover
+         color: rgb(28,170,152)
+
+.content--contact
+    margin-left: 20px
+    margin-right: 20px
+    paddin-top: 0
+    padding-bottom: 50px
+    padding-right: 10px
+    padding-left: 10px
+
+.form
+    margin-left: 20px
+    margin-right: 20px
+    padding: 50px 10px
+    padding-bottom: 0px
+
+
+    &__title
+        font-family: Arial, Helvetica, sans-serif
+        padding-bottom: 5px
+    
+    &__input
+        font-family: Arial, Helvetica, sans-serif
+        margin-bottom: 10px
+        border: solid lightgrey 1px
+        border-radius: 0.3em
+        padding: 5px
+        width: 100%
+
+.about
+    margin-left: 20px
+    margin-right: 20px
+    padding: 50px 10px
+    padding-bottom: 20px
+
+    &__collap
+        border: grey 1px solid
+        border-radius: 0.25em
+        margin: 10px auto
+        
+    
+    &__collap-title
+        padding: 10px
+        margin-bottom: 10px
+        border: lightgrey 1px solid
+        background-color: rgb(245, 245, 245)
+
+    &__collap-text
+        padding: 10px
+    
+    &__collap-text--1, &__collap-text--2, &__collap-text--3
+        display: none
+    
+    &__collap-title--1:hover + &__collap-text--1
```

Comment:
What's the use of the +? Is that the same as a ,?

Reply:
It is used to say: When you have hover to `&__collap-title--1` do things to `&__collap-text--1`.

In this case the text is not shown and when you pase the mouse over the title, the text is unfolded.

https://stackoverflow.com/questions/6910049/on-a-css-hover-event-can-i-change-another-divs-styling